package.json

```diff
@@ -17,10 +17,11 @@
   "contributors": ["Yuri Shkuro <ys@uber.com>", "Won Jun Jang <wjang@uber.com>"],
   "dependencies": {
     "node-int64": "^0.4.0",
-    "thriftrw": "^3.5.0",
-    "xorshift": "^0.2.0",
     "opentracing": "^0.13.0",
-    "uuid": "^3.2.1"
+    "prom-client": "^11.0.0",
```

Comment:
this isn't a required runtime dependency

Reply:
The newer version of npm installs by default. This has been removed and the Makefile updated to not save the dependency when it's installed.